spec/features/posts_spec.rb

```diff
@@ -0,0 +1,92 @@
+# rubocop:disable Metrics/BlockLength
+require 'rails_helper'
+
+RSpec.feature 'Posts', type: :feature do
```

Comment:
- [ ] **[OPTIONAL]** You added the post index and show page tests in this one file. I think it would have been best to separate them into different files for better code organization.